c++/src/Statistics.hh

```diff
@@ -41,49 +41,181 @@ namespace orc {
   };
 
 /**
+ * Internal Statistics Implementation
+ */
+
+  template <typename T>
+  class InternalStatisticsImpl {
+  private:
+    bool hasNull_;
```

Comment:
This is according to the google coding style we agreed upon. I am going to slowly change the entire code base to follow the google coding style. The style recommends class variables to end with an `_`
https://google.github.io/styleguide/cppguide.html#Variable_Names